labelbox/schema/slice.py

```diff
@@ -57,3 +58,40 @@ def get_data_row_ids(self) -> PaginatedCollection:
             dereferencing=['getDataRowIdsBySavedQuery', 'nodes'],
             obj_class=lambda _, data_row_id: data_row_id,
             cursor_path=['getDataRowIdsBySavedQuery', 'pageInfo', 'endCursor'])
+
+
+class ModelSlice(Slice):
+    """
+    Represents a Slice used for filtering data rows in Model.
+    """
+
+    def get_data_row_ids(self) -> PaginatedCollection:
+        """
+        Fetches all data row ids that match this Slice
+
+        Returns:
+            A PaginatedCollection of data row ids
+        """
+        query_str = """
+            query getDataRowIdsBySavedQueryPyApi($id: ID!, $from: String, $first: Int!) {
+                getDataRowIdsBySavedQuery(input: {
+                    savedQueryId: $id,
+                    after: $from
+                    first: $first
+                }) {
+                    totalCount
+                    nodes
+                    pageInfo {
+                        endCursor
+                        hasNextPage
+                    }
+                }
+            }
+        """
+        return PaginatedCollection(
+            client=self.client,
+            query=query_str,
+            params={'id': self.uid},
+            dereferencing=['getDataRowIdsBySavedQuery', 'nodes'],
+            obj_class=lambda _, data_row_id: data_row_id,
```

Comment:
not following here... according to PaginatedCollection documentation, obj_class is a class of an object...

Reply:
That documentation is 4 years old, I think it's outdated. I was actually just copying the code from CatalogSlice here 😅